cxx-qt-gen/test_inputs/passthrough.rs

```diff
@@ -21,6 +21,22 @@ pub mod my_object {
     // ItemForeignMod
     extern "C" {}
 
+    #[namespace = "namespace"]
+    extern "C" {}
+
+    #[namespace = "namespace"]
+    #[namespace = "namespace"]
```

Comment:
Repeating the same name twice? Are you checking here that two attributes on consecutive lines are properly handled? If so, I think using two different attributes would make the test easier to understand.

Reply:
Yup, i'm checking that multiple attributes are passed through, i'll rename one of them.

Reply:
Fixed.